optimism/src/mips/witness.rs

```diff
@@ -47,7 +47,7 @@ pub struct Env<Fp> {
     pub instruction_parts: InstructionParts<u32>,
     pub instruction_counter: usize,
     pub memory: Vec<(u32, Vec<u8>)>,
```

Comment:
Wasn't the content of the memory and registers of type `Variable`? Or does this casting only happen inside the interpreter?

Reply:
The `Variable` type is only in the interpreter, yea.